src/ansys/fluent/core/solver/flobject.py

```diff
@@ -679,56 +679,62 @@ def _get_new_keywords(obj, kwds):
     return newkwds
 
 
-class Command(Base):
-    """Command object."""
+class IBase(Base):
```

Comment:
The I prefix generally means interface so your intention is lost - anyway, better to call it something specific such as 
Action.

Reply:
Done. Thank you.